src/events.js

```diff
@@ -20,7 +20,23 @@ module.exports = function(ctx) {
 
   var events = {};
   var currentModeName = Constants.modes.SIMPLE_SELECT;
-  var currentMode = ModeHandler(modes.simple_select(ctx), ctx);
+  var currentMode;
+
+  function initialize() {
+    // Add event listeners
+    ctx.map.on('mousemove', events.mousemove);
+
+    ctx.map.on('mousedown', events.mousedown);
+    ctx.map.on('mouseup', events.mouseup);
+
+    if (ctx.options.keybindings) {
+      ctx.container.addEventListener('keydown', events.keydown);
+      ctx.container.addEventListener('keyup', events.keyup);
+    }
+
+    // Start us in simple_select mode
+    currentMode = ModeHandler(modes.simple_select(ctx), ctx);
```

Comment:
I'm pretty sure this is happening as the mode was connecting to the DOM. modes shouldn't connect this way so I think we can go back to how it was before.

I only want to go back to how it was before because I'm not 100% on what this change might mean. Also, I think that have an initialize function in what is effectively an initialize function is a bit confusing.